app/tasks/launch.js

```diff
@@ -105,6 +114,46 @@ let self = {
     return self.launch(exe_path, [], opts)
   },
 
+  launch_html_cave: async function(opts, cave) {
```

Comment:
I would probably feel better about having this in another module — launch already does so much.

I'm not sure what the right granularity would be, maybe `launch.js`, `launch/native.js` and `launch/html.js` ?